### PR TITLE
Add migration guide and update API reference for AEPMedia

### PR DESCRIPTION
**Pull request by cacheung**
Add migration guide and Update API reference for AEPMedia
Also make a fix to mobile-core-api-reference format.

### CODE REVIEW

File: using-mobile-extensions/adobe-media-analytics/media-api-reference.md [hunk 1]
```diff
@@ -2270,7 +3318,47 @@ _tracker.trackSessionStart(mediaObject, null);
 ```
 {% endtab %}
 
-{% tab title="iOS" %}
+{% tab title="iOS (AEP 3.x)" %}
+
+#### Media Resume
```
Comment (calebk1) [inline using-mobile-extensions/adobe-media-analytics/media-api-reference.md:3323]: Media resume.

File: using-mobile-extensions/adobe-media-analytics/migration.md [hunk 2]
```diff
@@ -0,0 +1,947 @@
+# Migrating to AEPMedia reference
+
+This document is a reference comparison of AEPMedia \(3.x\) APIs against their equivalent ACPMedia \(2.x\) APIs.
```
Comment (calebk1) [inline using-mobile-extensions/adobe-media-analytics/migration.md:3]: We don't need the "\\" to escape the parentheses. You can get rid of them.

File: using-mobile-extensions/adobe-media-analytics/migration.md [hunk 3]
```diff
@@ -0,0 +1,947 @@
...
+
+The AEPMedia extension is implemented purely in Swift and is compatible with the AEPCore swift SDK. To ensure a smooth transition from the ACPMedia SDK, there are no major changes on the API names or definition. For more details, follow the migration guide below for your Swift or Objective-C mobile application.  If explanation beyond showing API differences is necessary, it will be captured as an info hint within that API's section.
```
Comment (calebk1) [inline using-mobile-extensions/adobe-media-analytics/migration.md:5]: "... with the AEPCore Swift SDK."

File: using-mobile-extensions/adobe-media-analytics/migration.md [hunk 4]
```diff
@@ -0,0 +1,947 @@
...
+
+## Public classes
```
Comment (calebk1) [inline using-mobile-extensions/adobe-media-analytics/migration.md:7]: It feels a little weird to say "Public classes" here when, by definition, these docs are public. I'd go with "AEPMedia classes", or something like that instead.

File: using-mobile-extensions/adobe-media-analytics/migration.md [hunk 5]
```diff
@@ -0,0 +1,947 @@
...
+
+| Type          | AEP 3.x (Swift) | AEP 3.x (Objective-C) | ACP 2.x (Objective-C) |
+| ------------- | :-------------- | --------------------- | --------------------- |
+| Primary Class | Media           | AEPMobileMedia        | ACPMedia              |
+
+
+
+## Public APIs
```
Comment (calebk1) [inline using-mobile-extensions/adobe-media-analytics/migration.md:15]: Same comment here. It feels a little weird to say "Public APIs" here when, by definition, these docs are public. I'd go with "AEPMedia APIs", or something like that instead.

File: using-mobile-extensions/adobe-media-analytics/migration.md [hunk 6]
```diff
@@ -0,0 +1,947 @@
...
+```objective-c
+FOUNDATION_EXPORT NSString* _Nonnull const ACPAdMetadataKeyAdvertiser;
+```
+
+{% endtab %}
+
+{% endtabs %}
+
+{% hint style="info" %}
+For the full list of constant type, refer to [Media API reference](media-api-reference#standard-ad-constants).
```
Comment (calebk1) [inline using-mobile-extensions/adobe-media-analytics/migration.md:812]: Does the link like this work properly?

**Review by calebk1 — CHANGES_REQUESTED** [review]
A few comments that need to be done throughout the docs:

1. In a lot of the places, it's written as "Objective C", when it should be "Objective-C".
2. In a lot of places, you've written the parentheses with "\\". We don't need any of those, so I'd get rid of them.

File: using-mobile-extensions/adobe-media-analytics/migration.md (hunk 6 above)
Comment (cacheung) [reply using-mobile-extensions/adobe-media-analytics/migration.md:812]: It should work. I will also test once it is merged.

**Comment by cacheung** [comment]
> A few comments that need to be done throughout the docs:
> 
> 1. In a lot of the places, it's written as "Objective C", when it should be "Objective-C".
> 2. In a lot of places, you've written the parentheses with "\". We don't need any of those, so I'd get rid of them.

@calebk1  Updated review comments. Thanks.

File: using-mobile-extensions/adobe-media-analytics/media-api-reference.md [hunk 7]
```diff
@@ -76,7 +94,40 @@ Media.createTracker(new AdobeCallback<MediaTracker>() {
 ```
 {% endtab %}
 
-{% tab title="iOS" %}
+{% tab title="iOS (AEP 3.x)" %}
+
+#### createTracker
+
+Creates a media tracker instance that tracks the playback session. The tracker created should be used to track the streaming content, and it sends periodic pings to the media analytics backend.
```
[the remark this answers is not in the document]
Comment (calebk1) [reply using-mobile-extensions/adobe-media-analytics/media-api-reference.md:101]: Just missed this comment here!

[the remark this answers is not in the document]
Comment (cacheung) [reply using-mobile-extensions/adobe-media-analytics/media-api-reference.md]: Oh yeah, missed this one.  Updated.

**Review by calebk1 — APPROVED** [review]
LGTM!

**Review by praveek — APPROVED** [review]

**Pull request merged**